sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4100,13 +4099,8 @@ object functions {
    * @group collection_funcs
    * @since 2.3.0
    */
-  def from_json(e: Column, schema: String, options: Map[String, String]): Column = {
-    val dataType = parseTypeWithFallback(
-      schema,
-      DataType.fromJson,
-      "Cannot parse the schema in JSON format: ",
-      fallbackParser = DataType.fromDDL)
-    from_json(e, dataType, options)
+  def from_json(e: Column, schema: String, options: Map[String, String]): Column = withExpr {
+    new JsonToStructs(e.expr, lit(schema).expr, options)
```

Comment:
Could you update `ExpressionDescription` (usage and examples) of `CsvToStructs` and `JsonToStructs`?